ynput/ayon-core
# Fix: dirmap wasn't using local site values

## Changelog Description
Some DCC(Maya, Nuke) offer dirmap functionality, eg. remapping file paths from source to destination. If dirmap is enabled and artist is using local site, it should use as a source root value from `Site Settings` (which might be different for each artist) rather than static mapping from `ayon+settings://nuke/dirmap?project=ayon_test` for example.


## Testing notes:
1. enable dirmap in Nuke in `ayon+settings://nuke/dirmap?project=ayon_test`
2. you might keep mapping there empty, values from `Local Settings` should be used
3. open Nuke and look for 
![image](https://github.com/ynput/ayon-core/assets/4457962/02a8adf2-0d86-4178-bee7-c5ec05292fdd)
where values in `active roots overrides` should match what artist has in `Site Settings`
![image](https://github.com/ynput/ayon-core/assets/4457962/6bfcf7d0-6ac9-4e19-8981-be0cdc298858)
